### DISCUSSION TITLE
Long branch name breaks mobile PR layout

### DISCUSSION THREAD

**Discussion by michaelblyons** [community/community#29592] [Pull Requests]
Let's say I make a PR where the name of my actual branch is long (`pr/some-category/something-long-and-descriptive`). The layout of the whole page on mobile is widened (and scaled down) to accommodate the branch name. There's a place at the ~~bottom~~ _[Edit: top]_ of the page where the CSS does not allow break-word on the branch name.

**Comment by aconsuegra** [comment]
Hey @michaelblyons, thanks for your feedback! Are you using the mobile native app (if yes, could you tell me in which platform?) or the mobile web?

**Reply by michaelblyons** [reply]
Mobile website. Firefox for Android.

**Reply by aconsuegra** [reply]
Thanks for following up! We use the "Mobile" category for discussions related to the GitHub mobile apps only. I'll transfer this discussion to the "Pull Requests" category where someone will be able to assist you.
